tests/test_itrf_converter.py

```diff
@@ -34,6 +38,9 @@ def test_transform_itrf():
     assert result.elevation.to_numpy()[0] == 1.0052761882543564
 
 
-def test__datetime_to_decimal_year():
+@pytest.mark.parametrize("timezone", ["America/Denver", "UTC"])
+def test__datetime_to_decimal_year(timezone):
+    os.environ["TZ"] = timezone
+    time.tzset()
```

Comment:
:star_struck: 

Is the environment permanently changed after this test, depending on which timezone was tested last? My mental model is that it would be. Maybe we can copy the environment, wrap the test in `try:`, and restore the env in `finally:`? Or maybe there's a builtin context manager for contextually changing the environment...

Reply:
Good catch. 9518610